packages/babel-plugin-proposal-private-methods/test/fixtures/accessors/updates/input.js

```diff
@@ -43,5 +43,19 @@ class Cl {
 
     this.#privateFieldValue = -(this.#privateFieldValue ** this.#privateFieldValue);
     this.publicFieldValue = -(this.publicFieldValue ** this.publicFieldValue);
+
+    this.#privateField = 0n;
```

Comment:
Can you split this new test into a new one (`updates-bigint`) and add `"minNodeVersion": "10.0.0"` to its `options.json` file? Because older versions don't support bigints.